internal/lock/package.go

```diff
@@ -43,9 +65,29 @@ func (p *Package) IsAllowInsecure() bool {
 	return p.AllowInsecure
 }
 
+// Useful for debugging when we print the struct
+func (i *SystemInfo) String() string {
+	return fmt.Sprintf("SystemInfo{Outputs:%v, StorePath:%s}", i.Outputs, i.StorePath)
+}
+
+func (i *SystemInfo) DefaultStorePath() string {
+	if i == nil || len(i.Outputs) == 0 {
+		return ""
+	}
+
+	for _, output := range i.Outputs {
+		if output.Default {
+			return output.Path
+		}
+	}
+
+	return i.Outputs[0].Path
+}
+
 func (i *SystemInfo) Equals(other *SystemInfo) bool {
 	if i == nil || other == nil {
 		return i == other
 	}
-	return *i == *other
+
+	return slices.Equal(i.Outputs, other.Outputs)
```

Comment:
Does this need to take `i.StorePath` into account?

Reply:
I think this is OK? at least looking at the callsite https://github.com/jetpack-io/devbox/blob/main/internal/devbox/update.go#L142-L146

Only outputs matter moving forward and storePath is just kept for legacy reasons (to avoid updating everyone's lockfile)